audiomentations/augmentations/transforms.py

```diff
@@ -1074,7 +1086,7 @@ def apply(self, samples, sample_rate):
             noise_gain[-fade_out_mask.shape[0] :] = np.minimum(
                 noise_gain[-fade_out_mask.shape[0] :], fade_out_mask
             )
-            noise_samples = noise_samples * noise_gain
+            noise_samples = noise_samples * noise_gain  # Gain here describes just the gain from the fade in and fade out.
```

Comment:
Line is too long. I recommend putting the comment above.

I also recommend checking out the python formatting tool called `black`, which we use in this project. I can help you set this up in your IDE 😄 

Black recommends a max line length of 90-ish characters.